.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,27 @@
+name: Release Charts
+
+on:
+  push:
```

Comment:
I don't really understand this. Are we pushing new helm-charts versions after we push in this repo but when do we update the docker image (after pushing to hathor-core repo) we automatically update the image the chart is using?

Reply:
We are not setting up this automation to update the default hathor-core image used by the chart (the `appVersion` parameter in `Chart.yaml`)

This pipeline only publishes new versions of the chart if it detects the `version` in `Chart.yaml` has changed.

However, the user can override the hathor-core image when installing the chart, so he/she will not be prevented from upgrading his/her full-node just because we didn't automated this.